### ISSUE TITLE
Two configurations raising errors in `search.path_finder`. 

### ISSUE THREAD

**Issue by Luke-Poeppel** [Luke-Poeppel/decitala#101]
```
search.path_finder(
    "/Users/lukepoeppel/Messiaen/Oiseaux_De_Nouvelle_Calédonie/11_Les_Siffleurs/B_Le_Siffleur_à_ventre_roux/XML/Le_Siffleur_à_ventre_roux_Ex98.xml",
    0,
    "decitala"
)
```
```
search.path_finder(
   "/Users/lukepoeppel/Messiaen/Oiseaux_De_Nouvelle_Calédonie/1_Le_Rossignol_à_ventre_jaune/A_Le_Rossignol_à_ventre_jaune/XML/Le_Rossignol_à_ventre_jaune_Ex2.xml"
    0,
    "decitala",
    ignore_single_anga_class_fragments=True
)
```

**Comment by Luke-Poeppel** [comment]
Le Coucou à Eventail (Ex60) has the same UnboundLocalError as the first of the above.

**Comment by Luke-Poeppel** [comment]
Fixed in 5b4a082441536ccd9b80e336cb2575d1619510e4.
